package.json

```diff
@@ -33,6 +33,8 @@
     "opn": "^1.0.0",
     "serve-index": "^1.1.4",
     "serve-static": "^1.4.0",
-    "wiredep": "^2.0.0"
+    "wiredep": "^2.0.0",
+    "gulp-mocha": "~2.0.0",
```

Comment:
ugh. I was playing around with test-stuff, and then unwound all my stuff because the generated webapp seemed to already have it. Unfortunately, it seems I was testing against stale files in my tmp folder or something (that were put there by my gulp scripts, which I have since killed). blergh.

Will try to dive in tomorrow to figure it out. In the meanwhile, ship-it (by I somewhat prefer chai vs should)